packages/sdk-react/src/components/auth/Auth.tsx

```diff
@@ -129,26 +109,26 @@ const Auth: React.FC<AuthProps> = ({
         !getVerifiedSuborgsResponse ||
         !getVerifiedSuborgsResponse.organizationIds
       ) {
-        onError("Failed to fetch account");
+        onError(AUTH_ERRORS.SUBORG.FETCH_FAILED);
       }
       suborgId = getVerifiedSuborgsResponse?.organizationIds[0];
     } else {
       const getSuborgsResponse = await getSuborgs({ filterType, filterValue });
       if (!getSuborgsResponse || !getSuborgsResponse.organizationIds) {
-        onError("Failed to fetch account");
+        onError(AUTH_ERRORS.SUBORG.FETCH_FAILED);
```

Comment:
just curious: any particular reason to use snake case here over camel like other enums?